Bottom: ef5a60e573027f22c928b8eb4e5e46cb5154b3ab
Top:    6f9a0ef4f09cc9d876e1331b391cc8d830962428
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 20:32:00 +0000

Avoid clearing pins / vias from polygons... it is already done!



---

diff --git a/src/draw.c b/src/draw.c
index 0384f62..38abe43 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1239,6 +1239,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1261,6 +1262,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
